content/300-guides/200-deployment/201-serverless/400-deploy-to-aws-lambda.mdx

```diff
@@ -1,301 +1,213 @@
 
-Your directory contains the following files:
+In the `generator` block of your `schema.prisma` file, add the following:
 
-```terminal no-copy
-README.md
-handlers
-node_modules
-package-lock.json
-package.json
-prisma
-schema.sql
-serverless.yml
+```prisma file=schema.prisma
+binaryTargets = ["native", "linux-arm64-openssl-1.0.x"]
 ```
 
-## 2. Set the DATABASE_URL in <inlinecode>.env</inlinecode>
+## Loading environment variables via a <inlinecode>.env</inlinecode> file
 
-The Lambda functions need the `DATABASE_URL` environment variable to access the database. You must define an `.env` file. The preconfigured `serverless-dotenv-plugin` uses this file to inject into the function runtime.
+Your functions will need the `DATABASE_URL` environment variable to access the database. The `serverless-dotenv-plugin` will allow you to use your `.env` file in your deployments.
 
-The repository contains an `.env.example` example file to help with this.
+First, make sure that the plugin is installed
 
-### Steps
+```sh
+npm install -D serverless-dotenv-plugin
+```
 
-1. Enter the following command to copy `.env.example` to a new file, `.env`:
+Then, add `serverless-dotenv-plugin` to your list of plugins in `serverless.yml`
 
-   ```terminal
-   cp .env.example .env
-   ```
+```code file=serverless.yml no-copy
+plugins:
+  - serverless-dotenv-plugin
+```
 
-2. Open `.env` and set the `DATABASE_URL` to your database credentials. The format is as follows:
+No further configuration needed! Your environment variables defined in your `.env` file will now be loaded on package or deployment.
 
-   ```file
-   DATABASE_URL="postgres://__USER__:__PASSWORD__@__HOST__/__DATABASE__"
-   ```
+```sh no-copy highlight=3-4;normal
+❯ serverless package
+Running "serverless" from node_modules
+DOTENV: Loading environment variables from .env:
+         - DATABASE_URL
 
-   Replace the uppercase placeholders with your database credentials. For example:
+Packaging deployment-example-sls for stage dev (us-east-1)
+.
+.
+.
+```
 
-   ```file
-   DATABASE_URL="postgres://janedoe:randompassword@yourpostgres.supabase.co:6543/yourdbname"
-   ```
+## Connection pooling
 
-   > **Note:** If your connection URL contains any special characters, then you must [percentage encode them](/reference/database-reference/connection-urls#special-characters).
+Generally, when you use a FaaS (Function as a Service) environment to interact with a database, every function invocation can result in a new connection to the database. This is not a problem with a constantly running Node.js server. Therefore, it is beneficial to pool DB connections to get better performance. To solve this issue, you can use the [Data Proxy](/data-platform/data-proxy). For other solutions, see the [connection management guide for serverless environments](/guides/performance-and-optimization/connection-management#serverless-environments-faas).
```

Comment:
```suggestion
Generally, when you use a FaaS (Function as a Service) environment to interact with a database, every function invocation can result in a new connection to the database. This is not a problem with a constantly running Node.js server. Therefore, it is beneficial to pool DB connections to get better performance. You can use the [Accelerate](/data-platform/accelerate) to solve this issue. For other solutions, see the [connection management guide for serverless environments](/guides/performance-and-optimization/connection-management#serverless-environments-faas).
```

This is optional. It might be an opportunity to get feedback from Product on what we should recommend to our users.

Reply:
Let's figure that out separately, and not in this PR. 
(Optimally we just link to an article that is specifically for this problem and is easily kept up to date)